packages/components/src/intl/src/useLocalizedString.tsx

```diff
@@ -0,0 +1,29 @@
+import { useLocalizedStringFormatter } from "react-aria";
+
+import resourcesEnUS from "../intl/en-US.json";
+import resourcesFrCa from "../intl/fr-CA.json";
+
+const Resources = {
+    "en-US": resourcesEnUS,
+    "fr-CA": resourcesFrCa
+};
+
+export function useLocalizedString(): LocalizedStringFormatter {
+    return useLocalizedStringFormatter(Resources, "@hopper-ui/components");
+}
+
+/**
+ * LocalizedStringFormatter is not exported from react-aria, so we have to define it here.
```

Comment:
I think you mentioned this was fixed in react-aria now

Reply:
they release once a month sadly. so in the next 10 days hopefully!